src/mbgl/plugin/plugin_layer_impl.cpp

```diff
@@ -0,0 +1,118 @@
+#include "plugin_layer_impl.hpp"
+#include <iostream>
```

Comment:
imports need to be cleaned up,

use <mbgl.../plugin_layer_impl.hpp> for consistency